pkg/kn/commands/broker/describe_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
+
+import (
+	"errors"
+	"testing"
+
+	"gotest.tools/assert"
+	"gotest.tools/assert/cmp"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1beta1 "knative.dev/eventing/pkg/apis/eventing/v1beta1"
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+
+	clientv1beta1 "knative.dev/client/pkg/eventing/v1beta1"
+	"knative.dev/client/pkg/util"
+)
+
+func TestBrokerDescribe(t *testing.T) {
```

Comment:
Test is not complete... I can comment out following and still pass the test:

```go
// describeBroker print broker details to the provided output writer
func describeBroker(out io.Writer, broker *v1beta1.Broker, printDetails bool) error {
	dw := printers.NewPrefixWriter(out)
	commands.WriteMetadata(dw, &broker.ObjectMeta, printDetails)
	// dw.WriteLine()
	dw.WriteAttribute("Address", "").WriteAttribute("URL", broker.Status.Address.URL.String())
	// dw.WriteLine()
	// commands.WriteConditions(dw, broker.Status.Conditions, printDetails)
	if err := dw.Flush(); err != nil {
		return err
	}
	return nil
}
```

Reply:
Addressed in https://github.com/knative/client/pull/894/commits/25ec5bdc59f09f097812fb617e46b35bf1c53cb5, let me know if the test's acceptable now.